libs/pages/clusters/src/lib/ui/page-clusters-create/step-summary/step-summary.tsx

```diff
@@ -229,8 +246,64 @@ export function StepSummary(props: StepSummaryProps) {
             <div className="flex-grow mr-2">
               <div className="text-sm text-neutral-400 font-bold mb-2">Features</div>
               <ul className="text-neutral-350 text-sm list-none">
-                {Object.keys(props.featuresData).map((id: string) => {
-                  const currentFeature = props.featuresData && props.featuresData[id]
+                {props.featuresData.aws_existing_vpc && (
+                  <>
+                    <li className="mb-2">
+                      VPC ID:{' '}
+                      <strong className="font-medium">{props.featuresData.aws_existing_vpc.aws_vpc_eks_id}</strong>
+                    </li>
+                    <li className="mb-2">
+                      EKS subnets ids:{' '}
+                      <ul className="list-disc ml-4">
+                        {renderSubnets('Zone A:', 'A', props.featuresData.aws_existing_vpc.eks_subnets)}
```

Comment:
`renderSubnets` should be a component

Reply:
https://github.com/Qovery/console/pull/1192/commits/00b0d54afaaa4086856766dfd72b2bf967da544e